.seed-config.yaml

```diff
@@ -0,0 +1,108 @@
+project:
+  name: "Godocit"
+  state: "unstable"
+  image: "/blob/master/logo.svg"
+  website: "https://godocit.okkur.org"
+  oneLiner: "Create documentation needed issues across different repositories"
+  description: |
+    When a PR introduces a new feature that requires documentation, this action
+    can automatically create an issue on your website's repository to add
+    documentation for that new feature.
+  version: "v0.1.0"
+  mainLicense: "apache2"
+  docsLicense: "CCBySa"
+
+vision:
+  type: ""
+  overview: |
+    Provide an automatic flow for keeping track of documentation.
+  items:
+    - 'Works easily by just adding a "documentation needed" label on the PR'
+    - "Create issues on different repositories"
+  concept: "Godocit's core concept is to provide a automatic tool to never miss
+    documentation for new features."
+  aim: "Godocit provides an easy and fast way to create documentation issues for
+    new features and keep track of undocumented features"
+
+readme:
+  usageExample: |
+    For using godocit you first need to create a Github App and install it on your
+    user or organization that has read and write access to the source and target
+    repositories. For more information read the ["Authenticating with GitHub Apps"](https://developer.github.com/apps/building-github-apps/authenticating-with-github-apps/) page.
+
+    After creating the app and installing it on your user or organization, write the
+    following config in `.github/workflows/main.yml` file in your source repository:
+    ```
+    name: CI
+
+    on: 
+      pull_request:
+        types: [labeled]
+
+    jobs:
+      build:
+        runs-on: ubuntu-latest
+        steps:
+        - name: Create the documentation issues
+          uses: okkur/actions-godocit@v0.0.1
+          with:
+            targetRepo: 'USER/REPO'
+          env:
+            PRIVATE_KEY: ${{ secrets.PrivateKey }}
+            INSTALLATION_ID: ${{ secrets.InstallationID }}
+
+    ```
+
+    Note: Since in this version you should create a GitHub App for godocit, you 
+    have to replace the `uses: okkur/actions-godocit@v0.0.1` line with your
```

Comment:
Is this really necessary? It authenticates using the key as well as the installation ID so the rest shouldn't be specific to our app. So using the action is fine as long as their own app is provided or?

Reply:
That's right. My bad.